blueman/main/Sendto.py

```diff
@@ -271,9 +272,10 @@ def on_session_added(self, _manager, session_path):
 
     def on_session_removed(self, _manager, session_path):
         logging.debug('Session removed: %s' % session_path)
-        self.object_push.disconnect_by_func(self.on_transfer_started)
-        self.object_push.disconnect_by_func(self.on_transfer_failed)
-        self.object_push = None
+        if self.object_push:
```

Comment:
We never have a session removed without a session being added, why is there a need to check?